src/test/ui/nll/closure-requirements/propagate-approximated-to-static.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Test a case where we are forced to approximate one end-point with
+// `'static`. Note that `'static` shows up in the stderr output as `'0`.
+
+// compile-flags:-Znll -Zborrowck=mir -Zverbose
+
+#![feature(rustc_attrs)]
+
+use std::cell::Cell;
+
+// Callee knows that:
+//
+// 'x: 'a
```

Comment:
ok

Reply:
Actually, digging into this revealed something weird. Why is this not getting an error reported? It's propagating the right relationships, but they're not yielding any errors...

Reply:
Oh, it's another case of "is fixed later on in nll-master". In particular, it's https://github.com/rust-lang/rust/issues/45827 I think but maybe one of the other issues. I can add a fixme, but I just tested and I do get the expected errors on nll-master.